Src/can_wrapper/can_wrapper.c

```diff
@@ -243,6 +251,70 @@ static CANWrapper_StatusTypeDef transmit_internal(NodeID recipient, const CANMes
 	return status;
 }
 
+
+static CANWrapper_StatusTypeDef transmit_internal_raw(NodeID recipient, const CANMessage *msg, bool is_ack)
```

Comment:
Instead of passing `is_ack` and `recipient` as parameters, you can just use `msg->is_ack` and `msg->recipient`. So you don't need them. The extra `transmit_internal_raw` function also isn't needed.